src/sentry/api/endpoints/chunk.py

```diff
@@ -86,13 +86,14 @@ def post(self, request, organization):
         logger = logging.getLogger("sentry.files")
         logger.info("chunkupload.start")
 
-        files = request.data.getlist("file")
-        files += [GzipChunk(chunk) for chunk in request.data.getlist("file_gzip")]
-        if len(files) == 0:
+        if not request.data:
             # No files uploaded is ok
             logger.info("chunkupload.end", extra={"status": status.HTTP_200_OK})
             return Response(status=status.HTTP_200_OK)
 
+        files = request.data.getlist("file")
+        files += [GzipChunk(chunk) for chunk in request.data.getlist("file_gzip")]
```

Comment:
Wondering if it's worth early existing here if `files` is empty? Just in the case of other request params being passed.

Reply:
Oh, that'd preserve the way logs are now. Good catch, I didn't think about this.